environmentlogger/environmental_logger_json2netcdf.py

```diff
@@ -90,6 +90,12 @@
                     u'us': {"original":"microsecond", "SI":"second", "power":1e-6},
                     u'ppm': {"original":"pascal meter-2", "SI":"pascal meter-2", "power":1}, 
```

Comment:
umol / mol is dimensionless in the same way that ppm is, no? the CF 'canonical units' are mol/mol. 

In this case the numerator is the amount of CO2 and the denominator is the amount of air.